css/src/atomics/aspect-ratio.scss

```diff
@@ -0,0 +1,17 @@
+$aspect-ratios: (
+	'1-1': '1 / 1',
+	'2-1': '2 / 1',
+	'3-2': '3 / 2',
+	'4-3': '4 / 3',
+	'5-3': '5 / 3',
+	'9-16': '9 / 16',
+	'16-9': '16 / 9'
+) !default;
+
+@each $name, $ratio in $aspect-ratios {
+	.aspect-ratio-#{$name} {
+		aspect-ratio: #{$ratio};
```

Comment:
Atomics rule declarations should use `!important`

Reply:
updated